pom.xml

```diff
@@ -239,6 +239,12 @@
         <version>${google.core.version}</version>
         <type>test-jar</type>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core</artifactId>
+        <version>${google.core.version}</version>
+        <classifier>tests</classifier>
```

Comment:
Can you double check that this doesn't mess up the classifier downstream?

Reply:
I locally installed the artifact and tried in BQ and it's working as expected